src/main/java/com/google/api/gax/grpc/InstantiatingChannelProvider.java

```diff
@@ -177,31 +178,28 @@ String serviceHeader() {
 
   @VisibleForTesting
   static String getGaxVersion() {
-    String gaxVersion = DEFAULT_VERSION;
-    Properties gaxProperties = new Properties();
-    try {
-      gaxProperties.load(
-          InstantiatingChannelProvider.class
-              .getResourceAsStream("/com/google/api/gax/gax.properties"));
-      gaxVersion = gaxProperties.getProperty("version");
-    } catch (IOException e) {
-      e.printStackTrace(System.err);
-    }
-    return gaxVersion;
+    String gaxVersion = loadGaxProperty("version");
+    return gaxVersion != null ? gaxVersion : DEFAULT_VERSION;
```

Comment:
Seems like you can just return DEFAULT_VERSION from loadGaxProperty (if it is really same default version for both cases). I.e. in loadGrpcVersion():
```
catch (IOException e) {
    return DEVAULT_VERSION
}
```